sdk/ChangeLog.md

```diff
@@ -1,6 +1,63 @@
 InMobi Monetization SDK ChangeLog for Android
 =============================================
 
+## Build 10.7.7 [30/Aug/2024]
```

Comment:
Let's attach the website link of SDK changelogs(https://support.inmobi.com/monetize/sdk-documentation/android-guidelines/changelogs-android), instead of updating here to avoid the manual and repetitive task.
May be remove this file and attach the changelogs link in README